File: app/src/main/java/de/baumanngeorg/uvilsfrechner/service/uvi/model/DwdUviModel.java
package de.baumanngeorg.uvilsfrechner.service.uvi.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DwdUviModel {
    private String last_update;
    private String next_update;
    private String name;
    private String sender;

    private List<DwdUviContent> content;

    private String forecast_day;
}

File: app/src/main/java/de/baumanngeorg/uvilsfrechner/service/uvi/model/DwdUviModel.kt
package de.baumanngeorg.uvilsfrechner.service.uvi.model

data class DwdUviModel(
        val last_update: String = "2000-01-01T10:10:10",
        val next_update: String = "2000-01-01T10:10:10",
        val forecast_day: String = "XTX",
        val name: String = "XTX",
        val sender: String = "XTX",
        val content: List<DwdUviContent> = listOf(
                DwdUviContent(
                        city = "Berlin",
                        forecast = DwdUviForecast(
                                today = 8,
                                tomorrow = 8,
                                dayafter_to = 8
                        )
                )
        )
)